bugbug/repository.py

```diff
@@ -275,6 +282,20 @@ def get_revs(hg, date):
     return revs
 
 
+def get_directories(files):
+    if isinstance(files, str):
+        files = [files]
+
+    directories = set()
+    for path in files:
+        path_dirs = (
+            os.path.dirname(path).split("/")[:2] if os.path.dirname(path) else []
```

Comment:
Small improvement, you can limit the split works by passing maxsplit.
```suggestion
            os.path.dirname(path).split("/", 2)[:2] if os.path.dirname(path) else []
```

Do you have an example where `os.path.dirname(path)` returns a Falsy value?

Reply:
It returns false when just a file name is encountered in `commit.files`.

Example for this file `old-configure.in`, it returns an empty string.